restler/utils/logging/ndjson_serializer.py

```diff
@@ -120,10 +119,12 @@ def to_dict(self):
 
         tags.update(self.tags)
         tags.update(self.sequence_tags)
+        request_text = None if omit_request_text == True else self.request
+        
         return {
             'sent_timestamp': self.sent_timestamp,
             'received_timestamp': self.received_timestamp,
-            'request': self.request,
+            'request': request_text,
             'response': self.response,
             'request_json': None if self.request_json is None else json.dumps(self.request_json),
             'response_json': None if self.request_json is None else json.dumps(self.response_json),
```

Comment:
Unrelated to the feature in this PR, I think there might be a bug here:

`'response_json': None if self.request_json is None else json.dumps(self.response_json),`

Should it be:

`: None if self.response_json`